Clarified docker volumes (#50)
cpvalente/ontime-docs: src/content/docs/quick-tips/docker-image.mdx

* `4001:4001/tcp` binds local port 4001 to the same container port. Port 4001 is the default for the UI and is necessary to expose the app. 
* `8888:8888/udp` binds the localhost port 8888 (default for OSC input into the app) to the same in the container, so that it can receive data from the host machine targeted to the loopback interface.
* `8888:8888/udp` binds the localhost port 9999 (default for OSC output from the app) to the same in the container, this allows Ontime to send data out of the container.

> * `"./ontime-external:/external/"` expose the internal container directorie.

Reviewer: thank you @Punkerschaf ,

small typo here
```suggestion
* `"./ontime-external:/external/"` expose the internal container directory.
```

- `/external/styles/` contains the user provided CSS Override
- `/external/external/` is a placeholder directory that users can leverage to distribute data through Ontime 

> You can bind a single host directory to `/external/` for Ontime to store all its files in, or
> these three paths can be bound to individual host directories as in the sample docker-compose file:

Reviewer: suggestion, best not to mention the sample docker-compose file.
This could lead to confusion if the user doesnt know which file we mean or if we decide to edit the example in docker-hub

```suggestion
You can bind a single host directory to `/external/` for Ontime to store all its files in, or expose the individual parts to host directories:
```
